src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketCommitClientImpl.java

```diff
@@ -21,21 +26,42 @@ public BitbucketCommitClientImpl(BitbucketRequestExecutor bitbucketRequestExecut
     }
 
     @Override
+    @Nullable
```

Comment:
We need to annotate `BitbucketCommitClient` with the same annotation.
And this is my mistake- we use `@CheckForNull` as a rule in Jenkins, which I believe this is the correct annotation here :coneofshame: